### PR TITLE
Add a switch button to encode and decode raw instruction data between base58 and hex, fix explorer with custom rpc, fix adding custom rpc endpoint

### PR DESCRIPTION
**Pull request by josephlabeleven**
* Fix invalid instruction data when using raw data to send a transaction
* Include instruction index, instruction account index in error message when mapping internal transaction to web3 transaction
* Add switch button to encode and decode raw instruction data between base58 and hex
* Fix explorer url when using custom rpc network
* Redefine INetwork by replacing "local" with "custom"
* Add "custom" option to rcp network dropdown
* Fix endpoint url is not updating when adding custom rpc
* Avoid adding duplicate endpoint url

### CODE REVIEW

File: src/utils/state.ts [hunk 1]
```diff
@@ -16,6 +16,13 @@ import {
 import { newAccount, newInstruction } from "./internal";
 import { toSortableCollection } from "./sortable";
 
+export const RPC_NETWORK_OPTIONS: string[] = [
```
Comment (sohrab-) [inline src/utils/state.ts:19]: `: INetwork[]`

Comment (josephlabeleven) [reply src/utils/state.ts]: fixed

**Review by sohrab- — APPROVED** [review]
LGTM & LFG! Please remember to squash and give it a conventional commit message.

**Comment by josephlabeleven** [comment]
merged

**Pull request merged**